pyanaconda/ui/gui/spokes/lib/resize.glade

```diff
@@ -26,78 +28,78 @@
   </object>
   <object class="GtkAdjustment" id="resizeAdjustment">
     <property name="upper">100</property>
-    <property name="step_increment">1</property>
```

Comment:
These changes are unrelated. I guess these are autogenerated. Is there a reason for storing these here?

Reply:
Glade does this -- just opening and saving the file with a newer version will make some changes like these automatically. I guess the property names got changed between and 3.18 and 3.40...